src/cljs/rems/atoms.cljs

```diff
@@ -64,6 +64,9 @@
     :target :_new}
    title " " [external-link]])
 
+(defn enrich-user [user]
```

Comment:
is this really an atom? it's more like a domain util

Reply:
I think I will make it a component soon because for better formatting you can't use a plain string anymore.